### PR TITLE
Add post about the Cookie Store API.

### PR DESCRIPTION
**Pull request by pwnall**
What's changed, or what was fixed?
- New post describing the Cookie Store API, entering Origin Trials in Chrome 69

**Fixes:** N/A

**Target Live Date:** 2018-09-05

- [ ] This has been reviewed and approved by (NAME)
- [x] I have run `npm test` locally and all tests pass.
- [ ] I have added the appropriate `type-something` label.
- [x] I've staged the site and manually verified that my content displays correctly.

**CC:** @petele

### CODE REVIEW

**Comment by developit** [comment]
I am so excited for this API.

File: src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+`document.cookie`, which introduces jank and has a negative impact on battery
+life.
+
+The Cookie Store API brings an alternative method for observing cookie
+changes, which does not require polling.
+
+    cookieStore.addEventListener('change', (event) => {
+      for (const cookie in event.changed) {
+        if (cookie.name === 'session_id')
+          SessionCookieChanged(cookie.value);
```
Comment (developit) [inline src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md]: I believe convention here is to go with lowercase leading characters for function names.

File: src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+{# wf_featured_snippet: The Cookie Store API offers asynchronous acesss to HTTP cookies, and opens up the cookie jar to service workers. #}
+{# wf_blink_components: Blink>WebVR #}
+
+# Asynchronous Access to HTTP Cookies {: .page-title }
+
+{% include "web/_shared/contributors/victorcostan.html" %}
+
+The [Cookie Store API](https://www.chromestatus.com/feature/5658847691669504) is
+available for Origin Trials starting in Chrome 69. The API introduces the
+following exciting possibilities.
```
Comment (developit) [inline src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md]: `possibilities:`?

File: src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md [hunk 3]
```diff
@@ -0,0 +1,187 @@
+project_path: /web/_project.yaml
+book_path: /web/updates/_book.yaml
+description: The Cookie Store API offers asynchronous acesss to HTTP cookies, and opens up the cookie jar to service workers.
+
+{# wf_updated_on: 2018-09-01 #}
+{# wf_published_on: 2018-07-23 #}
+{# wf_tags: cookie,chrome69 #}
+{# wf_featured_image: /web/updates/images/generic/styles.png #}
+{# wf_featured_snippet: The Cookie Store API offers asynchronous acesss to HTTP cookies, and opens up the cookie jar to service workers. #}
```
Comment (developit) [inline src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md]: "asynchronous `access` to HTTP cookies"

File: src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+* Cookies can be accessed asynchronously, avoiding jank on the main thread.
+* Changes to cookies can be observed, avoiding polling.
+* Cookies can be accessed from service workers.
+
+## You (probably) don't need cookies
+
+Before diving into the new API, I'd like to state that cookies are still the Web
+platform's worst client-side storage primitive, and should still be used as a
+last resort. This isn't an accident - cookies are the Web's first client-side
```
Comment (developit) [inline src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md]: "cookies `were` the Web's first client-side"

File: src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md (hunk 1 above)
Comment (pwnall) [reply src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md]: Done

File: src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md (hunk 2 above)
Comment (pwnall) [reply src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md]: Done.

File: src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md (hunk 4 above)
Comment (pwnall) [reply src/content/en/updates/2018/09/asynchronous-access-to-http-cookies.md]: Done.

**Comment by pwnall** [comment]
@developit Thank you very much for the kind words and very useful feedback!

I've added the fixes as a separate commit. PTAL, and please let me know if you'd like me to squash.

**Pull request merged**